mne/decoding/base.py

```diff
@@ -71,37 +71,42 @@ def fit(self, X, y):
         ----------
         X : array, shape (n_samples, n_features)
             The training input samples to estimate the linear coefficients.
-        y : array, shape (n_samples,)
+        y : array, shape (n_samples,) or (n_samples, n_targets)
```

Comment:
more compact and consistent as `shape (n_samples,[ n_targets])` to show optional